File: Array/Rm_Duplica_Sorted_Arr.java
import java.util.Iterator;
import java.util.ArrayList;
//#include <iostream>
//using namespace std;
//int main(){
//	cout << "Hello World!" << endl;
//	return 0;
//}
public class Rm_Duplica_Sorted_Arr {
    public static void main(String[] args) {
        Rm_Duplica_Sorted_Arr rdsa = new Rm_Duplica_Sorted_Arr();
        int[] nums = {1,1,2,5,5,6,9,9};
        int length = rdsa.removeDuplicates(nums);
        System.out.println("result_array's length is : " + length);
    }
    public int removeDuplicates(int[] nums) {
        ArrayList<Integer> result = new ArrayList<Integer>();
        for(int i = 0;i < nums.length;i++){//array to arraylist
            result.add(nums[i]);
        }
        for(int j = 0;j < result.size()-1;j++){
            int key = result.get(j);
            for(int k = j+1;k < result.size();k++){
                if(key - result.get(k) == 0){
                    result.remove(k);
                    k--;
                }
            }
        }
        for(int m = 0;m < result.size();m++){
            nums[m] = result.get(m);
        }
        for(Iterator it = result.iterator();it.hasNext();){
            System.out.println(it.next());
        }
        return result.size();
    }
}

File: Array/Rm_Duplica_Sorted_Arr.cpp
#include <iostream>
#include <vector>
using namespace std;
class Rm_Duplica_Sorted_Arr{
    public:
        int removeDuplicates(vector<int>& nums){
            if(nums.empty()) return 0;
            int index = 0;
            for(int i = 1;i<nums.size();i++){
                if(nums[index] != nums[i]){
                    nums[++index] = nums[i];
                }
            }
            return index + 1;
        }
};

int main()
{
    Rm_Duplica_Sorted_Arr ans ;
    vector<int> nums;
    nums.push_back(1);
    nums.push_back(1);
    nums.push_back(2);
    int length = ans.removeDuplicates(nums);
    cout << length << endl;
    for(int i=0; i<length; i++)
        cout<<nums[i]<<" ";
    cout<<endl;
    return 0;
}